test/test_xpu.py

```diff
@@ -229,6 +230,13 @@ def test_events(self):
         stream.record_event(event)
         event.synchronize()
         self.assertTrue(event.query())
+        start_event = torch.xpu.Event(enable_timing=True)
+        end_event = torch.xpu.Event(enable_timing=True)
+        stream.record_event(start_event)
+        time.sleep(0.1)
+        stream.record_event(end_event)
```

Comment:
Since the precision is `ns`, we should add more `time` sensitive test cases.

Reply:
I am concerned that having too precise a time might block the CI frequently.